Makefile

```diff
@@ -69,14 +69,33 @@ run-kubernetes-master: stop-kubernetes-master
 	docker run \
 		--net=host --name st-controller-manager \
 		--detach \
-		gcr.io/google_containers/hyperkube-amd64:v${K8S_VERSION} \
+		gcr.io/google_containers/hyperkube-amd64:${K8S_VERSION} \
 		/hyperkube controller-manager \
                         --master=127.0.0.1:8080 \
                         --min-resync-period=3m \
                         --allocate-node-cidrs=true \
                         --cluster-cidr=10.10.0.0/16 \
                         --v=5
 
+	# Create CustomResourceDefinition (CRD) for Calico resources
+	# from the manifest crds.yaml 
+	docker run \
```

Comment:
We should simplify this by mounting in the manifests directory into the `st-apiserver` container and just calling out to `kubectl` from within.

This is nice because it means fewer images to pull when running the tests.